src/components/button-toggle/button-toggle.style.js

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+
+const StyledButtonToogle = styled.div`
+  display: inline;
+`;
+
+const StyledButtonToogleLabel = styled.label`
+  display: inline-block;
+  height: 40px;
+  padding: 0 24px;
+  border: 1px solid ${({ theme }) => theme.colors.border};
+  font-size: 14px;
+  font-weight: 600;
+
+  input:checked ~ & {
+    color: ${({ theme }) => theme.colors.white};
+    background-color: ${({ theme }) => theme.colors.tertiary};
+  }
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.whiteMix};
+    border-color: ${({ theme }) => theme.colors.tertiary};
+  }
+
+  &:focus {
+    outline: 3px solid ${({ theme }) => theme.colors.focus};
+  }
+
+  .content-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+  }
```

Comment:
that should be a style-component

Reply:
Resolving as being discussed elsewhere: https://github.com/Sage/carbon/pull/2021/files#r276644913